### ISSUE TITLE
Extract pages corrupts form

### ISSUE THREAD

**Issue by mdmcconnell** [pdfcpu/pdfcpu#1014]
I am running pdfcpu on the command line under wsl Ubuntu.  This behaviour is occuring under the most recent commit b9b3d0fca12d.

Using "pdfcpu extract -mode page f1040se.pdf" out on the first attached file yields the second attached.  The extracted page is no longer a valid form in some way.  First, Acrobat has the following error:
![image](https://github.com/user-attachments/assets/5eab2e6f-36af-4690-9858-80fa5207c752)

But more interestingly, pressing tab from any form field causes the reader to hang.
[f1040se.pdf](https://github.com/user-attachments/files/17919597/f1040se.pdf)
[f1040sep1.pdf](https://github.com/user-attachments/files/17919598/f1040sep1.pdf)

**Comment by hhrutter** [comment]
Well, seems like you are ripping your form apart..
Essentially what you want is `form splitting`, that's not implemented.
I also doubt it ever will be since there maybe form field dependencies between pages of your form containing fieds.

**Comment by mdmcconnell** [comment]
Yes, that's a good point - I did not know that this is possible, and have not seen it, but if there was some active element to the form and a cross dependency between fields, it would clearly be impossible separate those fields.  In the simple case of no dependency, I think it should be possible.  Or pdfcpu could throw an error instead of creating a (presumably invalid) PDF that crashes Acrobat.

**Comment by hhrutter** [comment]
Closed due to inactivity.
